src/library/simplified/extension/currency-code.js

```diff
@@ -65,6 +65,9 @@ else if (utag.isRailSearchResults() && b['entity.railSearch.searchResults.railLe
 else if (utag.isRailRateDetails() && b['entity.railSearch.railDetail.railLegs.0.railOfferItems.0.price.currency'] != undefined) {
     b['currencyCode'] = b['entity.railSearch.railDetail.railLegs.0.railOfferItems.0.price.currency'];
 }
+else if (utag.isRCO() || (b.utagPageName && b.utagPageName == "page.Checkout.Confirmation" && b.entity.productTypes && b.entity.productTypes.length == 1 && b.entity.productTypes[0] == "RAIL")){
+    b['currencyCode'] = b.entity.checkout.currency ? b.entity.checkout.currency : "";
```

Comment:
Was there recent work to set the currencyCode on other confirmation pages? Should we have a single condition to set the currency Code on all confirmation pages?

Reply:
it was for this same extension, and RCO was missed.

Reply:
I think you are correct, this entire extension can be replaced with just the following:

if(b.pageInfo.pageName && b.pageInfo.pageName.indexOf("Checkout.Confirmation") > -1 && b.entity.checkout.currency){
    b['currencyCode'] = b.entity.checkout.currency;
}